examples/with-tailwind/packages/ui/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "tailwindcss -i ./src/styles.css -o dist/index.css",
+    "build": "tailwindcss -i ./src/styles.css -o ./dist/index.css",
```

Comment:
Is this the bug fix?

Reply:
No it wasn't causing a bug. But the dev script was :
`"tailwindcss -i ./src/styles.css -o ./dist/index.css --watch"`
and the build script was :
`"tailwindcss -i ./src/styles.css -o dist/index.css"`
I changed the build script to match the way dev script was written.